### ISSUE TITLE
Enderite Shulker Box Hitboxes don't work like vanilla hitboxes

### ISSUE THREAD

**Issue by Umpaz** [Nic4Las/Minecraft-Enderite-Mod#20]
If you're standing on an enderite shulker box and open it you will fall through the block. In vanilla it will raise you up on the lid. If you are standing under an enderite shulker box and open it it just goes through you. In vanilla, it will lower you down into crouch mode.

**Comment by Nic4Las** [comment]
Good catch we will look into it. However we are currently working on updating the mod. That takes priority at the moment.
